Test/Machine/test_machine.py

```diff
@@ -31,8 +31,7 @@
 layers = [
     nk.layer.FullyConnected(
         input_size=g.n_sites,
-        output_size=40,
-        activation=nk.activation.Lncosh())
+        output_size=40)
```

Comment:
Shouldn't you add an activation layer here?

Reply:
yes, but maybe in the next PR we should add checks for all the implemented layers, just to make sure it's all OK!